libpod/stats.go

```diff
@@ -0,0 +1,126 @@
+package libpod
+
+import (
+	"path/filepath"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/containerd/cgroups"
+	"github.com/opencontainers/runc/libcontainer"
+	"github.com/pkg/errors"
+)
+
+// ContainerStats contains the statistics information for a running container
+type ContainerStats struct {
+	Container   string
```

Comment:
Can you name this ContainerID? Right now not obvious if it's name or ID

Reply:
done